libs/core/langchain_core/runnables/base.py

```diff
@@ -534,8 +534,6 @@ def get_config_jsonschema(
 
     def get_graph(self, config: Optional[RunnableConfig] = None) -> Graph:
         """Return a graph representation of this Runnable."""
-        from langchain_core.runnables.graph import Graph
```

Comment:
Looks like the module-level import was originally just for type checking; that was later changed but this import was not removed https://github.com/langchain-ai/langchain/commit/7d5800ee515bf0cf38b97d09abb6857b767ab05f